include/jdksavdecc_jdks.h

```diff
@@ -3,21 +3,16 @@
 /*
   Copyright (c) 2014, J.D. Koftinoff Software, Ltd.
   All rights reserved.
-
```

Comment:
All other header files have a space here. I don't see any reason to remove this spacing? The same is true of the other blank lines that have been removed. We should try to stick with the conventions of the other header files in this directory.

Reply:
This part of the files is retained. No changes in header from me. Anyways, we should adopt this change as well.

Reply:
I don't understand. git says there WAS a change in this part of the header file, otherwise there would not be any diff displayed. I might be mis-understanding what you are writing. Can you please elaborate?

Reply:
Same file in this git did not have header spacing. Anyways it has been taken now.